pkg/scheduler/apis/config/types.go

```diff
@@ -88,6 +88,9 @@ type KubeSchedulerConfiguration struct {
 	// then scheduler stops finding further feasible nodes once it finds 150 feasible ones.
 	// When the value is 0, default percentage (5%--50% based on the size of the cluster) of the
 	// nodes will be scored.
+	// Note: This field will be overridden by the profile-level PercentageOfNodesToScore when that is not zero.
+	// DEPRECATED: Please use the profile-level PercentageOfNodesToScore to configure this instead.
+	// TODO(#95446): Remove PercentageOfNodesToScore from KubeSchedulerConfiguration once v1beta1 is removed.
```

Comment:
I think we can delete this field now, we don't need it in the internal representation since we will directly default the per-profile field based on this one in defaults.go

Reply:
+1 to keep profile-pertentage only, particularly as Liggitt commented that obtaining value from another field doesn't work well.